LIMS_IMAGE/web/src/urls.py

```diff
@@ -16,13 +16,19 @@
 from django.contrib import admin
 from django.urls import path, include
 from django.views.generic.base import TemplateView
+from django.conf import settings
+from django.conf.urls.static import static
 
 # Django reads URL patterns from top to bottom
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('accounts/', include('accounts.urls')),
+    # TODO: why are there two accounts urls?
```

Comment:
Does there have to be two identical URL's for accounts?

Reply:
yes, the second one is the login pages, and the first one is the views from the accounts page
we want both to be under the accounts portion of the site

Reply:
Awesome, thanks for clarifying!